src/Shared/Helpers/OutputBuilder.cs

```diff
@@ -0,0 +1,190 @@
+﻿using Microsoft.CodeAnalysis.Sarif;
```

Comment:
This class could be better refactored into a factory, but since we have only two output formats in the foreseeable future, Ill just use a single class and function overloading